test/simplehashregistrar_test.js

```diff
@@ -102,35 +105,191 @@ describe('SimpleHashRegistrar', function() {
 					assert.equal(err, null, err);
 					assert.equal(result[0], 1); // status == Auction
 					assert.equal(result[1], 0); // deed == 0x00
-					// Expected to end 28 days from start
-					var expectedEnd = new Date().getTime() / 1000 + days(28);
+					// Expected to end 5 days from start
+					var expectedEnd = new Date().getTime() / 1000 + days(105);
 					assert.ok(Math.abs(result[2].toNumber() - expectedEnd) < 5); // registrationDate
 					assert.equal(result[3], 0); // value = 0
 					assert.equal(result[4], 0); // highestBid = 0
 					done();
 				});
 			},
-			// Advance to the 24th day
-			function(done) { advanceTime(days(22), done); },
+			// Advance 30 days
+			function(done) { advanceTime(days(30), done); },
+			// Advancing days only have an effect after a transaction
 			function(done) {
 				registrar.startAuction(web3.sha3('anothername'), {from: accounts[0]}, done);
-			},
-			// Check later auctions end 7 days after they start
+			},		
+			// Check later auctions end 5 days after they start
```

Comment:
There's no longer any reason in the code that the auction duration would depend on when you start it, though - so it seems to me that the test just adds clutter.